test/agent-policy-tests/test-update-utils/main.tf

```diff
@@ -14,19 +14,22 @@
  * limitations under the License.
  */
 
-provider "random" {
-  version = "~> 2.0"
-}
+####################################################################
+## Test module for the integration test that tests update behavior
+####################################################################
 
-resource "random_pet" "main" {
-  length    = 1
-  prefix    = "simple-example"
-  separator = "-"
+provider "google" {
+  version = "~> 2.0"
 }
 
-module "example" {
-  source = "../../../examples/simple_example"
-
-  project_id  = var.project_id
-  bucket_name = random_pet.main.id
+module "agent_policy_detailed" {
+  source       = "../../../modules/agent-policy"
+  project_id   = "${var.project_id}"
```

Comment:
Do these all need to be variables? They should mostly be hardcoded.

Reply:
Yeah, the handwritten integration tests manipulate `description`, `agent_rules`, `group_labels`, `os_types`, `zones`, and `instances`, so those need to be variables. As for `project_id` and `policy_id`, I think Joe wanted them to be passed into the bash script that runs the integration tests for easy maintenance.

Reply:
Ended up not needing this: moved handwritten integration tests + unit tests to use existing frameworks.